docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>I don't understand why you need this cell. As far as I can see, you are just renaming the dimensions here? GCM-Filters will work with <code>ke_og</code>,  <code>ds['ugos']</code>,  <code>ds['vgos']</code>  and the existing dimensions <code>latitude</code>,  <code>longitude</code> . In other words, renaming <code>latitude, longitude</code>  to <code>lat, lon</code>  is not necessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>